pkgs/development/interpreters/lua-5/default.nix

```diff
@@ -29,44 +25,36 @@ in rec {
       sed -e 's/ALL_T *= */& $(LUA_SO)/' -i src/Makefile
     '';
 
-    # a grep a
     postBuild = stdenv.lib.optionalString (!stdenv.isDarwin) ''
       set -x
```

Comment:
Is there a reason to keep verbosity here?

Reply:
absolutely none sorry :/

Reply:
Mwahaha that's what happens when you remove FIXME without looking around why it was there! (Not that it matters)

Reply:
Yep, I usually leave a TODO but the `a grep a` looked like a mispaste. It also happens when I work on a PR over longer than expected.
I believe it's ready to squash & merge. Not sure if it should go to staging or master though.

Reply:
arf it wasn't. Damn shell expansion rules ! reverted my cosmetic-induced bug.